registration-processor/pre-processor/registration-processor-packet-validator-stage/src/main/java/io/mosip/registration/processor/stages/packet/validator/PacketValidateProcessor.java

```diff
@@ -455,7 +457,41 @@ public MessageDTO process(MessageDTO object, String stageName) {
 
 	}
 
-		private boolean isValidSupervisorStatus(MessageDTO messageDTO) {
+	private void setPacketCreatedDateTime(InternalRegistrationStatusDto registrationStatusDto) throws ApisResourceAccessException, PacketManagerException, JsonProcessingException, IOException {
+		LogDescription description = null;
+		try {
+			description = new LogDescription();
+			Map<String, String> metaInfo = packetManagerService.getMetaInfo(
+					registrationId, registrationStatusDto.getRegistrationType(), ProviderStageName.PACKET_VALIDATOR);
+			DateTimeFormatter formatter = DateTimeFormatter.ofPattern(dateformat);
+			String packetCreatedDateTime = metaInfo.get(JsonConstant.CREATIONDATE);
+			if (packetCreatedDateTime != null && !packetCreatedDateTime.isEmpty()) {
+				LocalDateTime dateTime = DateUtils.parseToLocalDateTime(packetCreatedDateTime);
+				registrationStatusDto.setPacketCreateDateTime(dateTime);
+			} else {
+				description.setMessage(PlatformErrorMessages.RPR_PVM_PACKET_CREATED_DATE_TIME_EMPTY_OR_NULL.getMessage());
+				description.setCode(PlatformErrorMessages.RPR_PVM_PACKET_CREATED_DATE_TIME_EMPTY_OR_NULL.getCode());
+				regProcLogger.error(LoggerFileConstant.SESSIONID.toString(), LoggerFileConstant.REGISTRATIONID.toString(),
+						description.getCode() + " -- " + registrationId,
+						PlatformErrorMessages.RPR_PVM_PACKET_CREATED_DATE_TIME_EMPTY_OR_NULL.getMessage());
+			}
+		} catch (DateTimeParseException e) {
+			description.setMessage(PlatformErrorMessages.RPR_PVM_PACKET_CREATED_DATE_TIME_PARSE_EXCEPTION.getMessage());
+			description.setCode(PlatformErrorMessages.RPR_PVM_PACKET_CREATED_DATE_TIME_PARSE_EXCEPTION.getCode());
+			regProcLogger.error(LoggerFileConstant.SESSIONID.toString(), LoggerFileConstant.REGISTRATIONID.toString(),
+					description.getCode() + " -- " + registrationId,
+					PlatformErrorMessages.RPR_PVM_PACKET_CREATED_DATE_TIME_PARSE_EXCEPTION.getMessage() + e.getMessage());
+		}catch (IllegalArgumentException ex)
+		{
+			description.setMessage(PlatformErrorMessages.RPR_PVM_INVALID_ARGUMENT_EXCEPTION.getMessage());
+			description.setCode(PlatformErrorMessages.RPR_PVM_INVALID_ARGUMENT_EXCEPTION.getCode());
+			regProcLogger.error(LoggerFileConstant.SESSIONID.toString(), LoggerFileConstant.REGISTRATIONID.toString(),

```

Comment:
Since we don't stop the packet processing in all these exceptions in this method, can we rather have it as warn instead of error and have it without exception trace ?

Reply:
Changed the Log from error to warn. and stack trace is not displayed here.